Fortinet/fortigate/tests/DNS.STANDARD.json

```diff
@@ -48,12 +52,14 @@
     "observer": {
       "egress": {
         "interface": {
-          "name": "INTF-2"
+          "alias": "lan",
+          "name": "undefined"
```

Comment:
It seems that the changes in the parser override some existing values to replace them by an `undefined` value.
I suggest adding `undefined` in the `ignored_values` of the parser.